applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Details.js

```diff
@@ -293,9 +293,16 @@ $(function () {
             assessmentResultWidgetManager.refresh();            
         }
     );
+    function initCustomFieldCurrencies() {
+        setTimeout(function () {
+            $('.custom-currency-input').maskMoney();
+        }, 1000);
```

Comment:
this is always ominous and sounds like a race condition, what happens if we make this time 100ms instead?

Reply:
@AndreGAot - modified this code to use MutationObserver.